test/steps_test.go

```diff
@@ -2505,7 +2504,13 @@ func theDecodedTransactionShouldEqualTheOriginal() error {
 		return err
 	}
 
-	// direct tx equality checking isn't fully implemented in go-sdk so this test is incomplete
+	// This test isn't perfect as it's sensative to non-meaningfil changes (e.g. nil slice vs 0
```

Comment:
nit  - spelling preference : 
```suggestion
	// This test isn't perfect as it's sensitive to non-meaningful changes (e.g. nil slice vs 0
```

Reply:
Thanks!